Launcher.py

```diff
@@ -155,10 +155,10 @@ class Launcher(App):
         container: ContainerLayout
         grid: GridLayout
 
-        _tools = {c.display_name: c for c in components if c.type == Type.TOOL and isfile(get_exe(c)[-1])}
-        _clients = {c.display_name: c for c in components if c.type == Type.CLIENT and isfile(get_exe(c)[-1])}
-        _adjusters = {c.display_name: c for c in components if c.type == Type.ADJUSTER and isfile(get_exe(c)[-1])}
-        _funcs = {c.display_name: c for c in components if c.type == Type.FUNC}
+        _tools = {c.display_name: c for c in components if c.type == Type.TOOL}
+        _clients = {c.display_name: c for c in components if c.type == Type.CLIENT}
+        _adjusters = {c.display_name: c for c in components if c.type == Type.ADJUSTER}
+        _miscs = {c.display_name: c for c in components if c.type == Type.MISC}
```

Comment:
```suggestion
        _miscs = {c.display_name: c for c in components if c.type == Type.MISC and c.display_name}
```

> *  i don't think bool(display_name) should ever be False, so we could remove the None if ... and change the type hint for type to not be optional? if we want hide a button, it'd be better to use a special type or an additional flag rather than having display_name = ""

This was a wrong assumption. This was used in `worlds/LauncherComponents.py` to hide the first entry. We now get a blank button. Doing the filtering in __init__ feels jank though. Not sure if the proposed change is less jank, or if'd be better to introduce a `Type.HIDDEN`